yugabyte/yugabyte-db
# [#6540]  Platform: Remove certificate feature.

**Description**: Remove certificate feature which will be added in the list view of certificates to delete unused certificates.
If the certificate is linked to the existing universe then we will be disabling the button in the FE.

**Testing**: :

1. Go to certificates 
2. Try to delete the certificate which is not linked to any universe.
3. Also we can not see the delete option for certs which is linked to the universe.

**sbt run**
```
[info] Passed: Total 1186, Failed 0, Errors 0, Passed 1182, Skipped 4
```
